src/web/helpers/global-helper.ts

```diff
@@ -17,7 +17,6 @@ function getOS() {
 const setWindowMember = (): void => {
   window.os = getOS();
 };
-setWindowMember();
```

Comment:
If this line is removed, other files would not be able to get window.os when imported

Reply:
https://github.com/flux3dp/beam-studio/blob/3034ee6a412d22cf69d8d88a8fe069fd87f6c86c/src/main.ts#L16

https://github.com/flux3dp/beam-studio-web/blob/1a9a7e34afc01045ad28928cb969b54e6cc33a3a/src/index.tsx#L12

`setWindowMember` is explicitly invoked in `beam-studio` and `beam-studio-web`.